packages/flutter_tools/lib/src/cache.dart

```diff
@@ -896,6 +923,54 @@ class MacOSFuchsiaSDKArtifacts extends _FuchsiaSDKArtifacts {
   }
 }
 
+/// Abstract class for cached iOS/USB binary dependencies.
+abstract class IosUsbArtifacts extends CachedArtifact {
```

Comment:
Is there any way to handle all the ios usb artifacts together in this class rather than having a subclass-per-file?

Reply:
Done